.htaccess

```diff
@@ -0,0 +1 @@
+Deny from all
```

Comment:
This new file is mistake?

Reply:
I wanted to block direct access to WOVN.php directory.

Reply:
Ah, I see